src/scripts/__tests__/html-transforms.test.js

```diff
@@ -59,4 +60,33 @@ describe('HtmlTransforms module', () => {
     expect(document.querySelector('.wm-diff-other')).toBeInstanceOf(Element);
     expect(document.querySelector('.wm-diff-other')).toBeInstanceOf(Element);
   });
+
+  test('addTargetBlank adds a target attribute of "_blank" to only <a> tags', () => {
+    let document = parser.parseFromString(
+      `<!doctype html>
```

Comment:
Tiny comment: can you indent `<!doctype html>` to be consistent with the other tests above?